stable/spinnaker/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Open source, multi-cloud continuous delivery platform for releasing software changes with high velocity and confidence.
 name: spinnaker
-version: 1.2.0
+version: 1.2.1
```

adding new functionality should but the minor
```suggestion
version: 1.3.0
```